app/vmauth/README.md

```diff
@@ -280,6 +289,8 @@ See the docs at https://docs.victoriametrics.com/vmauth.html .
      Per-second limit on the number of WARN messages. If more than the given number of warns are emitted per second, then the remaining warns are suppressed. Zero values disable the rate limit
   -maxIdleConnsPerBackend int
      The maximum number of idle connections vmauth can open per each backend host (default 100)
+  -maxProxiedConnections int
+     The maximum number of connections that can be proxied by vmauth to backends (default 100)   
```

Comment:
```suggestion
     The maximum number of concurrent requests vmauth can proxy at any given time. Additional requests are rejected with 429 HTTP status code. There is no limit if set to 0. See also max_concurrent_requests option in per-user section at https://docs.victoriametrics.com/vmauth.html#auth-config (default 0)   
```

Reply:
moved this part of the text to the correct place according to changes